tests/Cargo.toml

```diff
@@ -27,6 +27,7 @@ pollster.workspace = true
 png.workspace = true
 wgpu.workspace = true
 wgt.workspace = true
+bytemuck.workspace = true
```

Comment:
Mind alphabetical order please

Reply:
👍 I fixed the order of all the dependencies in this file